x-pack/plugins/uptime/common/graphql/types.ts

```diff
@@ -32,6 +32,12 @@ export interface Query {
 
   getErrorsList?: (ErrorListItem | null)[] | null;
 }
+
+export interface PingResults {
+  total: UnsignedInteger;
+
+  pings?: Ping[] | null;
```

Comment:
I missed this when we were pairing, we forgot to require the list itself. See my comments on `pings/schema.gql.ts`.